### ISSUE TITLE
send to interactive w/ keyboard

### ISSUE THREAD

**Issue by ryan413** [Microsoft/PTVS#1984]
I'm trying to figure out what why can't i press CTRL+E, CTLR+E to send to interactive.. all i get is this light with a bing noise every time.

![image](https://cloud.githubusercontent.com/assets/18177650/21274408/adf16ec2-c38d-11e6-96d6-dd8ff7c56f79.png)

![image](https://cloud.githubusercontent.com/assets/18177650/21327929/0ef10e88-c5f7-11e6-8d62-9c722418831f.png)

EDIT:
version
![image](https://cloud.githubusercontent.com/assets/18177650/21327908/e59921c4-c5f6-11e6-924e-35e7e86d02a6.png)

**Comment by ryan413** [comment]
I guess try to reinstall... The other thing that is not working well is importing modules from same folder...  I don't know if I'm not doing something properly or what.  I added the folder to the search path, restarted computer, refreshed the intellisense, just deleted completions in local\pythontools, and also just deleted the python environment then recreated it in VS.  Am I missing something?

**Comment by ryan413** [comment]
Think I figured this out, I'll just call it operator error.  At some point I disabled the environment in the solution explorer and forgot about it.  Also, relative imports needed to make sure right environment is selected in the interactive terminal dropdown menu (not just environment window).  I'm closing this .

![image](https://cloud.githubusercontent.com/assets/18177650/21438356/135aea58-c84e-11e6-8919-2c908b39922d.png)

was only making this default environment, need to select from dropdown like image above.

![image](https://cloud.githubusercontent.com/assets/18177650/21438424/6a3e31fe-c84e-11e6-9c20-de91c3b874bf.png)
